src/app/feature/test/pages/tests/tests.component.scss

```
@@ -0,0 +1,254 @@
+}
+
+:host-context(html[data-mode='light']) .test-card__header,
+:host-context(html[data-mode='light']) .test-card__links {
+	border-color: #94a3b8;
+}
+
+.test-card__title {
+	font-size: 1.15rem;
+	font-weight: 700;
+	color: var(--c-text-secondary);
+	word-wrap: break-word;
```

Stylelint (17.7.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)